samples/spring-client-java-valueentity-counter/src/main/proto/com/example/actions/counter_states_sub.proto

```diff
@@ -0,0 +1,37 @@
+// Copyright 2021 Lightbend Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+// tag::state-subscription[]
+syntax = "proto3";
+package com.example.actions;
+
+import "akkaserverless/annotations.proto"; 
+import "com/example/domain/counter_domain.proto"; // <1>
+import "google/protobuf/empty.proto";
+
+option java_outer_classname = "StateSubscriptionApi";
+
+service CounterStateSubscription {
+  option (akkaserverless.codegen) = { 
+    action: {} // <2>
+  };
```

Comment:
same as above

Reply:
Removed akkaserverless annotations from all the proto files.